packages/blade/src/tokens/global/border.ts

```diff
@@ -7,6 +7,8 @@ type BorderRadius = Readonly<{
   medium: 4;
   /** large: 8(px/rem/pt) */
   large: 8;
+  /** xlarge: 12(px/rem/pt) */
+  xlarge: 12;
```

Comment:
Assuming this token is also now added on figma?

Reply:
There is an ongoing discussion. Will confirm this post that